internal/manager/setup.go

```diff
@@ -38,25 +37,19 @@ import (
 // -----------------------------------------------------------------------------
 
 // SetupLoggers sets up the loggers for the controller manager.
-func SetupLoggers(c *Config, output io.Writer) (logrus.FieldLogger, logr.Logger, error) {
-	deprecatedLogger, err := util.MakeLogger(c.LogLevel, c.LogFormat, output)
+func SetupLoggers(c *Config, output io.Writer) (logr.Logger, error) {
+	zapBase, err := util.MakeLogger(c.LogLevel, c.LogFormat, output)
 	if err != nil {
-		return nil, logr.Logger{}, fmt.Errorf("failed to make logger: %w", err)
+		return logr.Logger{}, fmt.Errorf("failed to make logger: %w", err)
 	}
-
-	if c.LogReduceRedundancy {
-		deprecatedLogger.Info("WARNING: log stifling has been enabled (experimental)")
-		deprecatedLogger = util.MakeDebugLoggerWithReducedRedudancy(output, &logrus.TextFormatter{}, 3, time.Second*30)
-	}
-
-	logger := logrusr.New(deprecatedLogger)
+	logger := zapr.NewLoggerWithOptions(zapBase, zapr.LogInfoLevel("v"))
```

Comment:
Maybe this line should be moved to `util.MakeLogger`? I think that we want rather always include this field. WDYT?

Reply:
Dunno. I just kept the original layout, where there was more of a distinction. MakeLogger created the underlying logging engine and returned it to SetupLoggers, and SetupLoggers then returned both that and a logr wrapper it created. Keeping the "build a thing that can be logr-ed" and "build a logr from some logr-able thing" code separate still kinda makes sense even if we now only return one.